src/fiatconnect/ReviewScreen.tsx

```diff
@@ -41,12 +41,41 @@ export default function FiatConnectReviewScreen({ route, navigation }: Props) {
     })
   }, [navigation])
 
+  useEffect(() => {
+    function hardwareBackPress() {
+      goBack()
+      return true
+    }
+    const backHandler = BackHandler.addEventListener('hardwareBackPress', hardwareBackPress)
+    return function cleanup() {
+      backHandler.remove()
+    }
+  }, [])
+
+  const goBack = () => {
```

Comment:
Could this just be part of `onPressBack`? Or do we not want analytics to fire on pressing hardware back?

Reply:
We don't fire events for hardware back button anywhere else in the app, so to be consistent I didn't think we should here.